### PR TITLE
[Netmanager] Map style component restructure

### PR DESCRIPTION
**Pull request by danielmarv**
#### Summary of Changes (What does this PR do?)

- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state

#### How should this be manually tested?

- This PR to restructure the Map style component because in it's initial stage it could obstruct the map
structure
- I have modified it so that it shows only the icon at default 
- when a user hovers on the icon it changes the color and it  brings a text "Change Map Mode"
- Then onclick it displays the options as shown below.

### CODE REVIEW

**Comment by Codebmk** [comment]
@danielmarv drop the text "change the map mode". Also the zoom effect on the map mode icon button doesn't seem to match the other icon buttons. It zooms out way more than them

File: netmanager/src/views/pages/Map/OverlayMap.js [hunk 1]
```diff
@@ -334,7 +369,7 @@ const CustomMapControl = ({
         onChange={onPollutantChange}
         showHeatMap={showHeatmap}
       />
-      <MapStyleSelector />
+      <MapStyleSelectorPlaceholder />
     </MapControllerPosition>
```
Comment (Codebmk) [inline netmanager/src/views/pages/Map/OverlayMap.js:373]: LGTM. Great work @danielmarv 

image

image

**Review by Codebmk — APPROVED** [review]

**Comment by Baalmart** [comment]
thanks @danielmarv for the AMAZING work!
Keep up this innovation spirit! The sky is not even the limit!
cc: @Codebmk

**Review by Baalmart — APPROVED** [review]
Thanks @danielmarv

**Pull request merged**